src/main/java/mate/academy/Main.java

```diff
@@ -6,14 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    public static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
+            logger.error("There is not an user with these login and password", e);
             e.printStackTrace();
```

Comment:
Using e.printStackTrace() is not recommended in production code as it prints to standard error and is not easily manageable with log files. It's better to fully rely on the logging framework.